### PR TITLE
Updates from bedrock

### PR DESCRIPTION
**Pull request by MozmarRobot**
From file changes in https://github.com/mozilla/bedrock/commit/85770605eb3cc8a92a49785212b28b1bbd12dc6e

### CODE REVIEW

File: en/firefox/more.ftl [hunk 1]
```diff
@@ -5,8 +5,9 @@
 ### URL: https://www-dev.allizom.org/firefox/more/
 
 firefox-products-are = { -brand-name-firefox } products are designed to protect your privacy
+learn-more-about-firefox = Learn more about { -brand-name-firefox }, it’s history, features and mission
```
Comment (peiying2) [inline en/firefox/more.ftl:8]: `it's` -> `its`

File: en/firefox/more.ftl [hunk 2]
```diff
@@ -22,6 +23,14 @@ get-the-privacy = Get the privacy you deserve. Enhanced Tracking Protection make
 take-your-passwords = Take your passwords everywhere
 securely-access-the = Securely access the passwords you’ve saved in { -brand-name-firefox } from anywhere — even outside of the browser.
 choose-which-firefox = Choose which { -brand-name-firefox-browser } to download in your language
+firefox-more-firefox-chromebook = { -brand-name-firefox-browser } for Chromebook
```
Comment (peiying2) [inline en/firefox/more.ftl:26]: `Chromebook` is a brand

File: en/firefox/more.ftl [hunk 3]
```diff
@@ -22,6 +23,14 @@ get-the-privacy = Get the privacy you deserve. Enhanced Tracking Protection make
...
+while-on-chromebook = While a Chromebook already has { -brand-name-chrome } installed, downloading and using Firefox as your go-to browser provides you with a few benefits
```
Comment (peiying2) [inline en/firefox/more.ftl:27]: `Chromebook`, `Firefox` are brands

File: en/firefox/more.ftl [hunk 4]
```diff
@@ -22,6 +23,14 @@ get-the-privacy = Get the privacy you deserve. Enhanced Tracking Protection make
...
+firefox-quantum = Firefox Quantum
```
Comment (peiying2) [inline en/firefox/more.ftl:28]: Both are brands

File: en/firefox/more.ftl [hunk 5]
```diff
@@ -22,6 +23,14 @@ get-the-privacy = Get the privacy you deserve. Enhanced Tracking Protection make
...
+quantum-was-revolution = Firefox Quantum was a revolution in Firefox development. In 2017, we created a new, lightning fast browser that constantly improves. Firefox Quantum is the { -brand-name-firefox-browser }.
```
Comment (peiying2) [inline en/firefox/more.ftl:29]: `Firefox` and `Quantum` are both brands

File: en/firefox/whatsnew/whatsnew-fx83.ftl [hunk 6]
```diff
@@ -0,0 +1,25 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
```
Comment (peiying2) [inline en/firefox/whatsnew/whatsnew-fx83.ftl:1]: Please remove this one from the PR. It is only for German.

**Review by peiying2 — CHANGES_REQUESTED** [review]
@amychurchwell - new challenge, when a few of you are working on different PRs, the automated script includes all outstanding PRs, including ones that should not be part of the l10n PR. 

- Missed a few brands in the PR
- Quantum file keeps showing up in the PR. Let's remove it in the PR.
- WNP#83 is included, which is for German only. Let's move it in the PR.

**Comment by peiying2** [comment]
@amychurchwell before I proceed, I want to confirm that the string IDs are correct. I see a mixture of filename prefix and lack there of.

**Comment by amychurchwell** [comment]
@peiying2 all new ID's should be prefixed correctly & I have removed the whatsnew page file.

**Review by peiying2 — COMMENTED** [review]
@amychurchwell I will go through the book.ftl file separately.

Given the length of this page, we are discussing with Justin on how to keep a content heavy page like this one to the target markets and offer an alternative, a shorter version to smaller communities. This page is best left for the l10n vendor to do. 

If you remove the book.ftl from this PR,  I will approve it.

**Comment by alexgibson** [comment]
> If you remove the book.ftl from this PR, I will approve it.

@peiying2 I believe the EU team already have translations for DE and FR done and waiting just as soon as this is available in Pontoon. Couldn’t we configure the .ftl file to only be available in those locales? E.g. https://github.com/mozilla/bedrock/blob/master/l10n/configs/pontoon.toml#L136

**Comment by peiying2** [comment]
> > If you remove the book.ftl from this PR, I will approve it.
> 
> @peiying2 I believe the EU team already have translations for DE and FR done and waiting just as soon as this is available in Pontoon. Couldn’t we configure the .ftl file to only be available in those locales? E.g. https://github.com/mozilla/bedrock/blob/master/l10n/configs/pontoon.toml#L136

I am all for the temporary solution. Let's do that. This is just too much for any communities to go through.

File: en/firefox/privacy/book.ftl [hunk 7]
```diff
@@ -0,0 +1,208 @@
...
+privacy-book-de-blog-post = If you want to learn more about the latter, here’s <a href="https://blog.mozilla.org/firefox/how-to-spot-real-fake-news/">an interesting blog post</a> that can be helpful.
+
+privacy-book-disable-unnecessary-location = Disable (unnecessary) location services:
+
+privacy-book-todays-mobile-devices = Today’s mobile devices typically have the ability to determine, store and share their location. That’s convenient for navigation or when you’ve lost your phone. At the same time though, this quite sensitive piece of information may also be used by apps, even if you don’t want it.
+
+privacy-book-many-social-media = <strong>Decide case by case:</strong> Many social media apps offer to share your location when you post something. It’s up to you if you want others to know where you are and it’s totally fine to say no.
+
+privacy-book-for-each-of = <strong>Decide for each app:</strong> For each of the apps on your device that wants access to your location information, you can decide if you want to share it. All major operating systems allow for you to change your settings so that you limit access for individual or all apps.
```
Comment (peiying2) [inline en/firefox/privacy/book.ftl:182]: Remove `for` in `... operating systems allow for you to change...`

File: en/firefox/privacy/book.ftl [hunk 8]
```diff
@@ -0,0 +1,208 @@
...
+
+privacy-book-another-reason-to = <strong>Avoid background localization:</strong> Another reason to do so is that some apps don’t only save your location when you use them – many of them also run the location service in the background and may create complete motion profiles of you over time. The app vendor may then use this information for a lot of purposes, even sell it, which may not suit you.
```
Comment (peiying2) [inline en/firefox/privacy/book.ftl:184]: `... some apps don’t only save your location ...` change `don't only save` to `not only save`.

File: en/firefox/privacy/book.ftl [hunk 9]
```diff
@@ -0,0 +1,208 @@
...
+
+privacy-book-dont-need-it = Don’t need it? Delete it!
+
+privacy-book-even-if-you = Even if you bought a smartphone with lots of memory so you’d never have to delete anything ever again, cleaning it out from time to time isn’t a bad idea. Why?
+
+privacy-book-the-more-apps = <strong>Limit the weakpoints.</strong> The more apps you have on your device, the more entities have access to your data – and it’s impossible to have the full overview all the time.
+
+privacy-book-check-which-apps = <strong>Spring-clean your phone:</strong> Check which apps you have installed on your device and if you’re still using them. Delete the ones (and any related accounts) you actually don't need anymore.
+
+privacy-book-also-reconsider-the = <strong>Limit the access.</strong> Also, reconsider the permissions you once gave to the remaining apps, including social networks, and ask yourselves: Does this app really need access to my location? My contacts? My photos? My camera and microphone?
```
Comment (peiying2) [inline en/firefox/privacy/book.ftl:194]: Change `yourselves` to singular `yourself`.

File: en/firefox/privacy/book.ftl [hunk 10]
```diff
@@ -0,0 +1,208 @@
...
+
+privacy-book-friendly-reminder-what = Friendly reminder: What are trackers again?
+
+privacy-book-we-distinguish-between = We distinguish between first and third-party trackers, or tracking cookies. You may find first-party trackers on news sites, for example. Their job is to notice when a new or recurring reader comes to the site, check out which articles they read, how much time they spend on each page, and so on. They may use this information to improve the quality of their offering and determine what readers actually like.
+
+privacy-book-thirdparty-trackers-on = Third-party trackers, on the other hand, may log a visitor's entire website visit, for example for the purpose of displaying targeted ads to them. This even works across different websites someone visits; plus, there’s rarely just one tracker on a site. They often work hand in hand with others to find new ways to convince you through ads to spend money, sign up for services, and then share more information about yourself.
```
Comment (peiying2) [inline en/firefox/privacy/book.ftl:200]: Insert a comma after example so it reads `... for example, for the purpose ...`

**Review by peiying2 — CHANGES_REQUESTED** [review]
@amychurchwell if you are to push all the pages in one PR, make sure to configure the book.ftl to `de`, `fr` only for now. Once we transition to the Smartling platform, we can enable the other locales. This page needs to go to paid services to complete.

**Comment by peiying2** [comment]
Let me know if there are updates on this one. If the `book` can be separated from the PR, we can get the other files pushed through.

File: en/firefox/privacy/book.ftl [hunk 11]
```diff
@@ -0,0 +1,208 @@
...
+privacy-book-tips-for = 15 tips <span>for a better digital life.</span>
+
+privacy-book-starting-over = Starting over:
+
+privacy-book-its-time-for = It’s time for a Data Detox! Together with the Tactical Tech Collective, { -brand-name-mozilla } has put together a nice little package of practical tips, called the Data Detox Kit. It’s a program that will make you feel like a whole new online person. Don't feel ready yet for the full program? Feel free to begin with one of the following tips instead (or all of them)!
+
+# Span is for visual formatting
+privacy-book-a-tidy-browser = A tidy browser, a tidy mind <span>– well, that may not be directly what “they” say but it’s still true!</span>
+
+privacy-book-delete-your-browser = <strong>Delete your browser history.</strong> Whether you’re using { -brand-name-firefox }, { -brand-name-chrome }, { -brand-name-safari }, or a different product – all of them offer this functionality. You’ll usually find it in the settings menu under "Security" or in a separate area. Here’s how to take care of it in { -brand-name-firefox }.
```
[the remark this answers is not in the document]
Comment (amychurchwell) [reply en/firefox/privacy/book.ftl]: A list of recommendations follows this, so a link is not necessary. Will update the grammar though. 👍

File: en/firefox/privacy/book.ftl [hunk 12]
```diff
@@ -0,0 +1,208 @@
...
+
+privacy-book-avoid-naming-your = <strong>Avoid naming your phone, tablet or computer after yourselves.</strong> It might seem convenient and easy to remember but it also helps others to recognize them as yours.
+
+privacy-book-disconnect-and-remove = <strong>Disconnect and remove all used public WIFIs once you don’t need them anymore.</strong> Otherwise, your devices may automatically reconnect to them whenever you’re nearby. You may not even notice it but it does create an unnecessary security risk.
+
+privacy-book-how-about-your = How about your Bluetooth?
+
+privacy-book-bluetooth-is-a = Bluetooth is a great invention that can save us a lot of cable clutter. However, most people don’t use it permanently. If you don’t own a smartwatch or another device that needs to be connected to your phone all the time, switch Bluetooth off. This can save battery power and, at the same time, removes a weak point intruders might make use of.
```
[the remark this answers is not in the document]
Comment (amychurchwell) [reply en/firefox/privacy/book.ftl]: I think `removes a weak point that intruders might make use of` might work better

**Comment by amychurchwell** [comment]
`book.ftl` is now in this PR: https://github.com/mozilla-l10n/www-l10n/pull/139
Recent changes from bedrock that were added to this PR are now here: https://github.com/mozilla-l10n/www-l10n/pull/140

**Comment by peiying2** [comment]
> `book.ftl` is now in this PR: #139
> Recent changes from bedrock that were added to this PR are now here: #140

Close the PR now.

**Pull request closed without merging**